tracer/src/Datadog.Trace/Logging/Internal/DatadogLogging.cs

```diff
@@ -218,5 +221,38 @@ internal static string GetLogDirectory()
 
             return logDirectory;
         }
+
+        internal static void CleanLogFiles(string logsDirectory)
+        {
+            var logDaysLimit = EnvironmentHelpers.GetEnvironmentVariable(ConfigurationKeys.LogFileRetentionDays);
+            if (int.TryParse(logDaysLimit, out var days) && (days > 0 && days < int.MaxValue))
```

Comment:
nit-picking, but `days` is an `int` so it can never be larger than `int.MaxValue`
```suggestion
            if (int.TryParse(logDaysLimit, out var days) && days > 0)
```